### ISSUE TITLE
list-workspaces --all with --format fails

### ISSUE THREAD

**Issue by exic** [nikitabobko/AeroSpace#453]
Hi!

```
❯ aerospace list-workspaces --all --format '%{workspace} %{monitor-id}'
--all conflicts with all other options
```

and

```
❯ aerospace list-workspaces --monitor all --format '%{workspace} %{monitor-id}'
1 2
3 1
7 2
...
```

should probably behave the same.

I'm trying to make sketchybar show only those workspaces for the respective monitor they're on, that's why I tried getting a list of all currently existing workspaces with their current monitor id.

## Checklist

- [x] I've searched for duplicates. My search queries were (please don't forget to search in closed issues too):
  - conflicts
- [x] I've read the documentation https://nikitabobko.github.io/AeroSpace/guide & https://nikitabobko.github.io/AeroSpace/commands
- [x] I've searched in the documentation. My search queries were:
  - format
  - all

```
aerospace CLI client version: 0.14.2-Beta 0cb8dbdfc5ee73b8cbc200f175f467ebead55201
AeroSpace.app server version: 0.14.2-Beta 0cb8dbdfc5ee73b8cbc200f175f467ebead55201
```

**Comment by nikitabobko** [comment]
Duplicate of #297

**Comment by nikitabobko** [comment]
I read it wrong. `--format` should indeed work together with `--all`. It's a bug

**Comment by nikitabobko** [comment]
Released in 0.15.0
